camping.py

```diff
@@ -143,6 +143,12 @@ def _main(parks):
         help="End date [YYYY-MM-DD]. You expect to leave this day, not stay the night.",
         type=valid_date,
     )
+    parser.add_argument(
+        "--nights",
+        default=0,
```

Comment:
I'd rather there not be a default and just treat the case where this is not set (it is None) as the "all nights" case.

Reply:
set nights=None by default